app/src/androidTest/java/com/github/sdpcoachme/SignupActivityTest.kt

```diff
@@ -67,4 +113,54 @@ open class SignupActivityTest {
         }
     }
 
+    @Test
+    fun afterDbExceptionUserStaysInSignUpActivity() {
```

Comment:
Very good that you thought of this failure case. Important to have that in the tests.